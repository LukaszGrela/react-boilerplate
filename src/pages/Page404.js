import React from 'react';
import {Link} from 'react-router-dom';

const Page404 = () => (
    <article>
        <h1>404</h1>
        <p>Can't find what you were after.</p>
        <Link to='/'>Home</Link>
    </article>
);

export default Page404;