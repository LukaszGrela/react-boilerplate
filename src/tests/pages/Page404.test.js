import React from 'react';
import {shallow} from 'enzyme';
import Page404 from '../../pages/Page404';

test('should render correctly', () => {
    const wrapper = shallow(<Page404 />);

    expect(wrapper).toMatchSnapshot();
})