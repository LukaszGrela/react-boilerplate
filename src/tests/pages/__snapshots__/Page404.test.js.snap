// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<article>
  <h1>
    404
  </h1>
  <p>
    Can't find what you were after.
  </p>
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</article>
`;
